src/wazuh_modules/vulnerability_scanner/src/policyManager/policyManager.hpp

```diff
@@ -685,14 +691,11 @@ class PolicyManager final : public Singleton<PolicyManager>
      * This function retrieves the current status of the manager's scan
      * from the configuration and returns it as a ManagerScanStatus enum value.
      *
-     * @return ManagerScanStatus The status of the manager's scan.
-     *         - If the scan is disabled, returns ManagerScanStatus::DISABLED.
-     *         - If the scan is enabled, returns ManagerScanStatus::ENABLED.
+     * @return true if the manager's scan is disabled, false otherwise.
      */
-    ManagerScanStatus getManagerDisabledScan() const
+    bool getManagerDisabledScan() const
```

Comment:
I understand the implementation but the name is not clear to me. What about isManagerScanDisabled()? or simillar

Reply:
Its following the naming of the other getters (get + name of the policy)